### DISCUSSION TITLE
scope charset support

### DISCUSSION THREAD

**Discussion by WeWebsterchen** [ory/hydra#2604] [Q&A] [answered]
now STRATEGIES_SCOPE=DEPRECATED_HIERARCHICAL_SCOPE_STRATEGY can support users matches users.read.own

how can i implement users matches users:read:own type scope。can i config this in environment variable?

**Comment by vinckr** [comment, marked as answer]
Can you provide some more context, ideally a place in the documentation that references what you are looking for?



> how can i implement users matches users:read:own type scope

Can you elaborate on this a bit more? What do you mean by `users:read:own`?

**Reply by WeWebsterchen** [reply]
this is sample，now when STRATEGIES_SCOPE=DEPRECATED_HIERARCHICAL_SCOPE_STRATEGY scope users can matches users.read.I want to use a colon as a delimiter, not a dot like users:read. can i config delimiter in hydra config file?
https://bitbucket.org/openid/fapi/src/master/Financial_API_Lodging_Intent.md
![image](https://user-images.githubusercontent.com/30589136/124456327-22f57080-ddbd-11eb-985e-1509477f7847.png)

**Reply by vinckr** [reply]
There was an issue about it a while back: https://github.com/ory/hydra/issues/1263 . 
So I don't think its possible right now, but a path for implementation is in the issue.
